docs/UserGuide.md

```diff
@@ -65,6 +65,94 @@ e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 * If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
 </box>
 
+--------------------------------------------------------------------------------------------------------------------
+### Adding a guest: `guest add`
+Adds a guest to the guest list.
```

Comment:
This differs from the `vendor add` description, which is "Adds a vendor to WedLog." I think it would be ideal if this could be standardized. Perhaps let's stick with "to WedLog"?